modules/manual-configuration-image-registry-ecr.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="manual-configuration-image-registry-ecr_{context}"]
+= Manually configuring Amazon Elastic Container Registry
+
+You can integrate {product-title} with Amazon Elastic Container Registry.
+
+.Prerequisites
+* You need an *Access Key ID* and a *Secret Access Key*. Alternatively, you can use a node-level IAM proxy such as `kiam` or `kube2iam`.
+* The access key must have read access to ECR.
+See https://aws.amazon.com/premiumsupport/knowledge-center/create-access-key/[How do I create an AWS access key?]  for more information.
+* If you are running {product-title} in Amazon Elastic Kubernetes Service (EKS) and want to integrate with an ECR from a separate Amazon account, you must first set a repository policy statement in your ECR.
+Follow the instructions at https://docs.aws.amazon.com/AmazonECR/latest/userguide/set-repository-policy.html[Setting a Repository Policy Statement]  and for *Actions*, choose the following scopes of the Amazon ECR API operations:
+
+** ecr:BatchCheckLayerAvailability
+** ecr:BatchGetImage
+** ecr:DescribeImages
+** ecr:GetDownloadUrlForLayer
+** ecr:ListImages
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Images* section, select *Google Container Registry*.
+. The *Configure image integration* modal box opens.
+Fill in the required details for:
+.. *Integration Name*: The name you want to give to this integration.
+.. *Types*: Select *Registry*.
+.. *Registry ID*: The ID of the registry.
+.. *Endpoint (Optional)*: The address of the registry.
+.. *Region*: The region for the registry.
+.. *Use Container IAM Role*: Turn on the toggele if you are using IAM.
+.. *Access Key ID (required if not using IAM)* and *Secret Access Key (required if not using IAM)*: Your access key and secret if you are not using IAM.
+. Select *Test* (checkmark icon) to test that the integration with the selected registry is working.
```

Comment:
Is it really a checkmark icon, or could this just say "checkmark"?

Reply:
Depending on the screen resolution the button looks like this:
![image](https://user-images.githubusercontent.com/23069445/123351081-bd74d900-d59f-11eb-921a-dcf35c000b5c.png)
OR
![image](https://user-images.githubusercontent.com/23069445/123351112-d2516c80-d59f-11eb-9ce5-0a0f23103d6d.png)

What do you suggest?

Reply:
That's fine then, I just wanted to make sure that it wasn't just a regular checkmark element